src/pages/Food/FoodModal/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import MainButton from 'components/MainButton';
+import SFoodModal from './style';
+
+function FoodModal() {
+  const [renderModal, setRenderModal] = useState(true);
+
+  const hideTheModal = () => {
+    setRenderModal(!renderModal);
+  };
+
+  return (
+    <>
+      {renderModal ? (
```

Comment:
you can replace this by renderModal && ... to gain some lines of code

Reply:
Done